lib/neek.js

```diff
@@ -56,14 +56,20 @@ module.exports = {
 
 };
 
-function isReadStream(rs){
-  return isStream(rs, process.stdin, '_read');
+function isReadStream(obj) {
+  // some streams are readable but do not have `read()`, however
+  // we can reliably detect readability by testing for `resume()`
+  return isStream(obj) && typeof obj.resume === 'function';
 }
 
-function isWriteStream(ws){
-  return isStream(ws, process.stdout, '_write');
+function isWriteStream(obj) {
+  return isStream(obj) && typeof obj.write === 'function';
 }
 
-function isStream(st, sys, prop){
-  return st && (st === sys|| typeof(st[prop]) === 'function' || typeof(st) === 'string');
-}
\ No newline at end of file
+function isStream(obj) {
+  return obj && typeof obj.pipe === 'function';
+}
+
+function isString(obj) {
```

Comment:
Can you replace this with `https://www.npmjs.com/package/lodash.isstring` please?